scripts/README.md

```diff
@@ -0,0 +1,15 @@
+# How to rebuild the gribi protos
+
+## Install dependencies
+
+* brew install coreutils
+* http://github.com/openconfig/ygot for proto_generator
```

Comment:
You can just do `go install github.com/openconfig/ygot/proto_generator@latest` and this will install this to your `GOPATH` (`~/go` by default).